pkg/corerp/renderers/gateway/render_test.go

```diff
@@ -1578,7 +1637,7 @@ func validateMetadata(t *testing.T, httpRoute *contourv1.HTTPProxy, metadataOpti
 }
 
 // createExpectedHTTPRouteSpec creates the expected HTTP route spec for validation.
-func createExpectedHTTPRouteSpec(routeName string, port int32, rewrite *contourv1.PathRewritePolicy, enableWebsockets bool) contourv1.HTTPProxySpec {
+func createExpectedHTTPRouteSpec(routeName string, port int32, rewrite *contourv1.PathRewritePolicy, timeout *contourv1.TimeoutPolicy, enableWebsockets bool) contourv1.HTTPProxySpec {
```

Comment:
Not in the scope of this PR but I think we should slowly move to the pattern of `With`. Like **CreateHTTPRouteSpec** can create the minimum HTTPRoute and if we do `.WithTimeoutPolicy` it would add the **TimeoutPolicy**. That would make things easier in cases like adding new params like this.